### PR TITLE
[ticket/15491] Fix out dated linkes in Installer support page

### PR DESCRIPTION
**Pull request by hubaishan**
Fix out dated linkes in Installer support page

PHPBB3-15491

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15491

### CODE REVIEW

File: phpBB/language/en/install.php [hunk 1]
```diff
@@ -49,7 +49,7 @@
...
... stable version</li><li>converting from phpBB 2.0.x to phpBB3</li><li>converting from other discussion board software to phpBB3 (please see the <a href="https://www.phpbb.com/community/viewforum.php?f=486">Convertors Forum</a>)</li></ul><p>We encourage users still running beta versions of phpBB3 to replace their installation with a fresh copy of the latest version.</p><h2>Extensions / Styles</h2><p>For issues relating to Extensions, please post in the appropriate <a href="https://www.phpbb.com/community/viewforum.php?f=451">Extensions Forum</a>.<br />For issues relating to styles, templates and themes, please post in the appropriate <a href="https://www.phpbb.com/community/viewforum.php?f=471">Styles Forum</a>.<br /><br />If your question relates to a specific package, please post directly in the topic dedicated to the package.</p><h2>Obtaining Support</h2><p><a href="https://www.phpbb.com/support/">Support Section</a><br /><a href="https://www.phpbb.com/support/docs/en/3.2/ug/quickstart/">Quick Start Guide</a><br /><br />To ensure you stay up to date with the latest news and releases, why not <a href="https://www.phpbb.com/support/">subscribe to our mailing list</a>?<br /><br />',
```
Comment (vinny) [inline phpBB/language/en/install.php]: @phpbb/support-team: does the **mailing list** still exist?

Comment (Noxwizard) [reply phpBB/language/en/install.php]: No, it doesn't.

Comment (hubaishan) [reply phpBB/language/en/install.php]: Delete the mailing list paragraph?
I suggest to change it phpBB twitter account.

Comment (marc1706) [reply phpBB/language/en/install.php]: Yes, though it makes sense to mention both our facebook page as well as twitter.

**Pull request merged**